dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/util/InputUtils.java

```diff
@@ -195,13 +207,13 @@ public CategoryOptionCombo getAttributeOptionCombo( CategoryCombo categoryCombo,
         {
             Set<CategoryOption> categoryOptions = new HashSet<>();
 
-            for ( String uid : options )
+            for ( String option : options )
```

Comment:
Could we optimize and get all the options in one SQL query?

Reply:
Yes possibly. Though wanted to check that all of them exists and give a warning if not for the specific ones.